dev-games/godot/metadata.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>rossbridger.cc@gmail.com</email>
+		<name>Ross Charles Campbell</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>games@gentoo.org</email>
+		<name>Gentoo Games Project</name>
+	</maintainer>
+	<longdescription>
+Godot Engine is a feature-packed, cross-platform game engine to create 2D and
+3D games from a unified interface. It provides a comprehensive set of common
+tools, so users can focus on making games without having to reinvent the wheel.
+Games can be exported in one click to a number of platforms, including the major
+desktop platforms (Linux, macOS, Windows) as well as mobile (Android, iOS) and
+web-based (HTML5) platforms.
+	</longdescription>
+	<use>
+		<flag name="bullet">Enable support for Bullet (Physics Engine)</flag>
+		<flag name="enet">Enable support for net-libs/enet</flag>
+		<flag name="lto">Enable support for link time optimization</flag>
+		<flag name="freetype">Enable support for media-libs/freetype</flag>
+		<flag name="mbedtls">Enable support for net-libs/mbedtls</flag>
```

Comment:
Use `<pkg>` tags for Gentoo packages mentioned in the flags. For example: `<flag name="mbedtls">Enable support for <pkg>net-libs/mbedtls</pkg></flag>`.